### PR TITLE
Liquibase script to clean the duplicate values

### PR DESCRIPTION
**Pull request by nitw-shobhit**
https://trello.com/c/DQhQJ1b8/2559-records-create-via-the-api-with-a-country-have-the-country-listed-twice-in-the-ui

### CODE REVIEW

File: orcid-persistence/src/main/resources/db/updates/clean-address-table.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    the software.
+
+    =============================================================================
+
+-->
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+        
+    <changeSet id="CLEAN-DUPLICATE-ADDRESSES" author="Shobhit Tyagi">
+        <sql>delete from address where id in (select id from address where id not in (SELECT min(id) FROM address group by iso2_country, orcid))</sql>
```
Comment (rcpeters) [inline orcid-persistence/src/main/resources/db/updates/clean-address-table.xml]: Nice job with the select min and grouping.

**Comment by amontenegro** [comment]
Hey @nitw-shobhit 

I think you can remove the second query ... isnt this: 

delete from address where id in (
     select id from address where id not in (
         SELECT min(id) FROM address group by iso2_country, orcid
     )
)

the same than this:

delete from address where id NOT IN (
     SELECT min(id) FROM address group by iso2_country, orcid
)

Please let me know if am wrong.

Thanks

**Comment by nitw-shobhit** [comment]
@amontenegro yes you are right. i am updating it now. thanks.

**Comment by nitw-shobhit** [comment]
@amontenegro  updated. thanks.

**Pull request merged**